lib/AppInfo/Application.php

```diff
@@ -64,10 +64,9 @@ public function register(IRegistrationContext $context): void {
 
 		$context->registerEventListener(LoadSidebar::class, LoadSidebarListener::class);
 		$context->registerEventListener(UserDeletedEvent::class, UserDeletedListener::class);
-	}
 
-	public function boot(IBootContext $context): void {
```

Comment:
Well, basically requires restoring this method. 🤔

Reply:
Ok... After restoring this method, I was able to test it, and everything seems to work properly. 😄 

Please, add that.. 😉

Reply:
Oops. I left part of the old code to compare the difference so I didn't catch that, my bad. :persevere: